src/gui/torrentcreatordialog.cpp

```diff
@@ -103,17 +103,27 @@ void TorrentCreatorDialog::updateInputPath(const Path &path)
     updateProgressBar(0);
 }
 
+#ifndef Q_OS_WIN
+// When the root file/directory of the created torrent is a symlink, we want to keep the symlink name in the torrent.
+constexpr QFileDialog::Options fileDialogOptions{QFileDialog::DontResolveSymlinks};
```

Comment:
I would prefer `const` as there is nothing to compute here:
```suggestion
const QFileDialog::Options fileDialogOptions {QFileDialog::DontResolveSymlinks};
```

Reply:
Done (also moved it to an anonymous namespace to avoid external linkage)